common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -21,6 +21,14 @@
         }
     }
 
+    @if(item.content.isPaidContent) {
+      <div class="content__section-label--advertisement">
+        <a class="content__section-label__link content__section-label__link--advertisement" href="@LinkTo {/@item.content.sectionLabelLink}">
+          @Html(Localisation(item.content.sectionLabelName))
+        </a>
+      </div>
+    }
```

Comment:
To be on the safe side, I'd set the next condition as an alternate:

`} else if(!(item.content.isImmersive && item.content.tags.isArticle)) {`